src/pages/Encounters/tabs/EncounterOverviewTab.tsx

```diff
@@ -58,8 +58,8 @@ export const EncounterOverviewTab = ({
         {/* Left Column - Symptoms, Diagnoses, and Questionnaire Responses */}
         <div className="flex-1 space-y-4" data-cy="encounter-overview">
           {canEdit && (
-            <div className="flex justify-between gap-2">
-              <div className="grid grid-cols-2 sm:grid-cols-3 gap-2 mx-auto md:mx-0">
+            <div className="flex flex-row justify-between gap-2">
```

Comment:
flex alone is just fine for horizontal right ? 

```suggestion
            <div className="flex justify-between gap-2">
```